polls/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from accounts.models import CustomUser
+
+# Create your models here.
+
+class Company(models.Model):
+    name = models.CharField(max_length=100)
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+class Poll(models.Model):
+    company = models.ForeignKey(Company, related_name='polls', on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    price = models.IntegerField()
+    category = models.ForeignKey(Category, on_delete=models.DO_NOTHING)
+    short_description = models.TextField(max_length=50)
+    description = models.TextField(max_length=500)
+    time_needed = models.TimeField()
+    rating = models.FloatField()
+    assigned_users = models.ManyToManyField(CustomUser, related_name='assigned_polls', through='PollAssignment')
+
+class PollAssignment(models.Model):
+    user = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
+    poll = models.ForeignKey(Poll, on_delete=models.CASCADE)
+    assigned_date = models.DateTimeField()
+    completed_date = models.DateTimeField(null=True)
+
+class Question(models.Model):
+    QUESTION_TYPES = [
+        (0, 'single_choice'),
+        (1, 'multi_choice'),
+        (2, 'text')
+    ]
+    poll = models.ForeignKey(Poll, related_name='questions', on_delete=models.CASCADE)
+    type = models.IntegerField(choices=QUESTION_TYPES)
+    text = models.TextField(max_length=500)
+
+class QuestionOption(models.Model):
```

Comment:
Chyba Ci tutaj uciekły dwa obiekty z bazy Options i CustomQuestionAnswers. O ile ten drugi bedziemy potrzebować przy odpowiadaniu na ankiety to pierwszy potrzebny nam jest już teraz.

Dodatkowo zwracam uwagę, że w Option nie powinno być ustawionej opcji on delete cascade, ponieważ opcje będą reużywane pomiędzy wriantami pytań.

Reply:
Options było jako text w QuestionOption, zmieniłem to na many to many, CustomQuestionAnswers to na razie nullowalny string, zmieni się później.